extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -163,5 +160,27 @@ amp-carousel .amp-carousel-button.amp-disabled {
   order: 100000000 !important;
   margin-right: -1px !important;
 }
-
 /* End SlideScroll */
+
+
+
+
+
+/* Begin ScrollableCarousel */
+.-amp-scrollable-carousel-container .amp-carousel-slide {
```

Comment:
the todo can be accomplished with just the slides public class - user can add a custom class like `.my-carousel`

and do `.my-carousel .amp-carousel-slide` to override.